dbms/src/Storages/KVStore/MultiRaft/RegionData.cpp

```diff
@@ -305,19 +300,22 @@ size_t RegionData::serialize(WriteBuffer & buf) const
 
 void RegionData::deserialize(ReadBuffer & buf, RegionData & region_data)
 {
-    size_t total_size = 0;
-    total_size += RegionDefaultCFData::deserialize(buf, region_data.default_cf);
-    total_size += RegionWriteCFData::deserialize(buf, region_data.write_cf);
-    total_size += RegionLockCFData::deserialize(buf, region_data.lock_cf);
+    RegionDataMemDiff size_changed;
 
-    region_data.cf_data_size = total_size;
-    reportAlloc(total_size);
+    size_changed.add(RegionDefaultCFData::deserialize(buf, region_data.default_cf));
+    size_changed.add(RegionWriteCFData::deserialize(buf, region_data.write_cf));
+    size_changed.add(RegionLockCFData::deserialize(buf, region_data.lock_cf));
+
+    region_data.cf_data_size = size_changed.payload;
+    region_data.decoded_data_size = size_changed.decoded;
+    recordMemChange(size_changed.payload);
```

Comment:
```suggestion
    region_data.recordMemChange(size_changed);
```
`region_data.cf_data_size` and `region_data.decoded_data_size` will be changed in `region_data.recordMemChange`

Reply:
ditto